packages/features/bookings/Booker/Booker.tsx

```diff
@@ -1,6 +1,10 @@
 import { AnimatePresence, LazyMotion, m } from "framer-motion";
```

Comment:
This should be a client component because you are using useState etc, no?

```suggestion
"use client";

import { AnimatePresence, LazyMotion, m } from "framer-motion";
```